apps/nrm/translations/src/en/pages.json

```diff
@@ -96,7 +96,10 @@
     "list-header": "Documents added"
   },
   "fr-details": {
-    "header": "Placeholder page"
+    "header": "What are your contact details?",
+    "read-only-field-1-label": "Email",
+    "read-only-field-1-hint": "This is the address you gave at the start of the form.",
+    "read-only-field-1-placeholder": "FR EMAIL PLACEHOLDER"
```

Comment:
are you going to do this later?

Reply:
Yes In MS-826 , We are we getting those values from? @sulthan-ahmed

Reply:
it'll be in the sessionModel something like `req.sessionModel.get('email')` We do this in right-to-rent-check somewhere